README.md

```diff
@@ -4,7 +4,10 @@ CSE 6550: Software Engineer Concepts, Fall 24
 
 California State University, San Bernardino
 ## Description
-The Textbook Chatbot project for CSE 6550 is designed to assist with queries related to the textbook "Software Engineering: A Practitioner's Approach." The chatbot serves as an educational tool, helping users by providing information, answering questions, and possibly retrieving content from the textbook.
+The Textbook Chatbot project for CSE 6550 is designed to assist with queries related to the textbook.
```

Comment:
I prefer to keep the old version and add the textbook reference/citation either at the end of the readme or below this section.